pkg/api/message/v1/service.go

```diff
@@ -324,9 +324,12 @@ func (s *Service) SubscribeAll(req *proto.SubscribeAllRequest, stream proto.Mess
 }
 
 func (s *Service) Query(ctx context.Context, req *proto.QueryRequest) (*proto.QueryResponse, error) {
-	log := s.log.Named("query").With(zap.Strings("content_topics", req.ContentTopics))
-	log.Debug("received request")
+	log := s.log.Named("query")
 	numContentTopics := len(req.ContentTopics)
+	if numContentTopics < 200 {
```

Comment:
I like this change, but did you mean to put it in this PR

Reply:
Yes, I figured it was small enough that it didn't matter too much. The tests were failing with unreadable logs before this change